lib/wallet/src/Cardano/Wallet/Write/Tx/Redeemers.hs

```diff
@@ -0,0 +1,232 @@
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TupleSections #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{- HLINT ignore "Use <$>" -}
+
+-- |
+-- Copyright: © 2023 IOHK
+-- License: Apache-2.0
+--
+-- Module containing 'assignScriptRedeemers'
```

Comment:
In my opinion, his comment still doesn't tell the reader very much: we could just as well delete this line. (A reader can establish that the module contains the `assignScriptRedeemers` function just by reading the export list. 😄)